impress-org/givewp
# Epic: GiveWP 3.0.0

## Description



This is the epic branch of GiveWP 3.0.0.

Todo:

Javascript related:
- [ ] Update all temporary `@ts-ignore` comments
- [x] Determine if can remove `.babelrc` file (currently commented out because conflicts with wp-scripts)

Release related:
- [x] Update since tags to 3.0.0
- [x] Update min WP version to 5.9
- [x] Update min WP version to 6.0
- [x] Update min php version to 7.2
- [ ] Migrate give to react 18
- [ ] Remove/update welcome banner from feature plugin
- [ ] Update any non 'give' textdomains (if present)
- [ ] Determine if should auto-activate test gateway (we were doing this in feature plugin)
- [ ] Search for any TODO comments
- [ ] Do NOT squash commits


## Affects



- Merged https://github.com/impress-org/givewp-next-gen

## Visuals



WIP

## Testing Instructions



If you currently have the feature plugin installed, you'll probably want to uninstall it before switching to this branch.  I did add a check in the feature plugin to prevent installing on givewp >= 3.0.0 but if it's already installed you'll likely run into some problems.

To build assets for the first time, run 

```
npm install --legacy-per-deps && npm run dev
```

Zip: https://github.com/impress-org/givewp/actions/runs/5871715750

## Pre-review Checklist
